content/working_with/official_plugins/Orchestration/helm.md

```diff
@@ -463,6 +463,52 @@ node_templates:
 {{< /highlight >}}
 
 
+# Workflows
+
+## update_repositories workflow
+
+This workflow provide the ability to update all the repositories for a Helm client.
+
+**Parameters:**
+
+* `node_instance_id` - Node instance ID of `cloudify.nodes.helm.Repo` node type which its helm client repos will be updated.
```

Comment:
* `node_instance_id` - Node instance ID of the `cloudify.nodes.helm.Repo` node type which its helm client repos will be updated.